cookbook/spec_add_ethereum.json

```diff
@@ -12,7 +12,7 @@
                 "block_distance_for_finalized_data": 64,
                 "blocks_in_finalization_proof": 3,
                 "average_block_time": "13000",
-                "allowed_block_lag_for_qos_sync": "5",
+                "allowed_block_lag_for_qos_sync": "1",
```

Comment:
1 is too few, might have edge cases with nodes that just updated and those that didnt